SnapMD.ConnectedCare.Sdk.Test/Properties/Settings.settings

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version='1.0' encoding='utf-8'?>
+<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)" GeneratedClassNamespace="SnapMD.ConnectedCare.Sdk.Tests" GeneratedClassName="Settings">
+  <Profiles />
+  <Settings>
+    <Setting Name="ApiDeveloperId" Type="System.String" Scope="Application">
+      <Value Profile="(Default)">HexaDecimal Api Developer Id Here</Value>
```

Comment:
Can you remove "HexaDecimal" as it is incorrect? It would be better to leave the node empty than to have invalid data here.